app/controllers/teachers/questions_controller.rb

```diff
@@ -0,0 +1,57 @@
+class Teachers::QuestionsController < ApplicationController
+  before_action :authenticate_user!
+  before_action :check_role
+
+  def index
+    @questions = Question.all
+  end
+
+  def show
+    @question = Question.find(params[:id])
+  end
+  def new
+	  @question = Question.new
+  end
+
+  def edit
+    @question = Question.find(params[:id])
+  end
+
+  def create
+    @question = Question.new(question_params)
+ 
+    if @question.save
+      redirect_to  [:teachers, :questions]
```

Comment:
be consistent, use same style ..
`redirect_to  [:teachers, :questions]`
or 
`redirect_to teachers_questions_path`